ServiceWebsite/ServiceWebsite/ClientApp/src/app/modules/shared/boolean.pipe.ts

```diff
@@ -0,0 +1,16 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
+@Pipe({
+  name: 'appBooleanHumanView'
+})
+
+export class AppBooleanHumanViewPipe implements PipeTransform {
```

Comment:
Test class for this? Doesn't need anything fancy but good to have some coverage for expectations

Reply:
done